bin/gltf-pipeline.js

```diff
@@ -1,114 +1,51 @@
 #!/usr/bin/env node
 'use strict';
 var fs = require('fs');
+var argv = require('yargs').argv;
 var path = require('path');
-var argv = require('minimist')(process.argv.slice(2));
-var addDefaults = require('../').addDefaults;
-var removeUnusedImages = require('../').removeUnusedImages;
-var removeUnusedSamplers = require('../').removeUnusedSamplers;
-var removeUnusedShaders = require('../').removeUnusedShaders;
-var removeUnusedTechniques = require('../').removeUnusedTechniques;
-var removeUnusedPrograms = require('../').removeUnusedPrograms;
-var removeUnusedBuffers = require('../').removeUnusedBuffers;
-var removeUnusedBufferViews = require('../').removeUnusedBufferViews;
-var removeUnusedMaterials = require('../').removeUnusedMaterials;
-var removeUnusedSkins = require('../').removeUnusedSkins;
-var removeUnusedCameras = require('../').removeUnusedCameras;
-var removeUnusedTextures = require('../').removeUnusedTextures;
-var removeUnusedMeshes = require('../').removeUnusedMeshes;
-var removeUnusedNodes = require('../').removeUnusedNodes;
-var removeUnusedAccessors = require('../').removeUnusedAccessors;
-var removeUnused = require('../').removeUnused;
-var loadGltfUris = require('../').loadGltfUris;
-var writeGltf = require('../').writeGltf;
-var parseBinaryGltf = require('../').parseBinaryGltf;
-var addPipelineExtras = require('../').addPipelineExtras;
-var convertDagToTree = require('../').convertDagToTree;
-var combineMeshes = require('../').combineMeshes;
-var combinePrimitives = require('../').combinePrimitives;
-var removeUnusedVertices = require('../').removeUnusedVertices;
-var OptimizationStatistics = require('../').OptimizationStatistics;
 var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
 var defined = Cesium.defined;
+var writeGltf = require('./writeGltf');
+var writeBinaryGltf = require('./writeBinaryGltf');
+var gltfPipeline = require('../lib/gltfPipeline');
 
 if (!defined(argv._[0]) || defined(argv.h) || defined(argv.help)) {
```

Comment:
Adding this comment again because the previous comment is hidden:

Now that we support -i for input files, argv._[0] may be undefined but shouldn't go the the help screen. As an alternative you can check for process.argv.length < 3.